src/less/documentation-swedbankpay.less

```diff
@@ -498,6 +498,123 @@
+                }
+            }
+
+            .tabs {
+                margin: 0;
+                position: absolute;
+                top: 0;
+                width: 100%;
+
+                &.tabs-fade-right {
+                    &:after {
+                        right: 56px; // width of options-open icon
+                        height: 39px;
+                    }
+                }
+
+                &.tabs-fade-left {
+                    &:before {
+                        height: 39px;
+                    }
+                }
+
+                ul {
+                    position: relative;
+                    width: 100%;
+
+                    &:before {
+                        content: "";
+                        position: absolute;
+                        top: 39px;
+                        width: 100%;
+                        border-bottom: 1px solid @brand-secondary-light-3;
+                    }
+                }
+
+                .options-open {
+                    display: flex;
+                    align-items: center;
+                    cursor: pointer;
+                    height: 40px;
+                    border-bottom: 1px solid @brand-secondary-light-3;
+                    padding: 0 1rem;
+
+                    &.hidden {
+                        visibility: hidden;
+                    }
+                }
+            }
+
+            .component-preview {
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                padding: 4.5rem 2rem 2rem;
+                border-bottom: 1px solid @brand-secondary-light-3;
+            }
+
+            .component-description {
+                padding: 2rem;
+                min-height: 15rem;
+                max-width: 40.5rem;
+            }
+
+            .options {
+                @media screen and (max-width: @screen-sm-max) {
+                    position: absolute;
+                    right: 0;
+                    height: 100%;
+                }
+
+                background-color: @white;
+                display: none;
+                min-width: 0;
+                border-left: 1px solid @brand-secondary-light-3;
+                transition: min-width 1s ease-in-out;
+                z-index: calc(@zindex-tabs + 2);
+
+                &.active {
+                    display: block;
+                    min-width: 12.5rem;
+                }
+
+                .options-header {
+                    padding: 0 1rem;
+                    display: flex;
+                    justify-content: space-between;
+                    align-items: center;
+                    border-bottom: 1px solid @brand-secondary-light-3;
+                    height: 2.5rem;
+
+                    .options-close {
+                        cursor: pointer;
+                    }
+                }
+
+                .options-body {
+                    padding: 1rem;
+                }
+            }
+        }
```

Comment:
Perhaps this class can be in a shared documentation file as we discussed on slack.

Reply:
"This class" here refers to `&.showcase-panel-advanced`